### DISCUSSION TITLE
Approvals

### DISCUSSION THREAD

**Discussion by jayb323** [outline/outline#3395] [Ideas]
Hey all,

Having a document approval workflow for ISO 9001 would be amazing. This is the only feature stopping it from being usable in true enterprise for some documents (policies, procedures, handbooks etc.).

I'd pay for this to be developed, BTW.

Cheers,

Jay

**Comment by tommoor** [comment]
Hey @jaybinder – this is on the roadmap, hopefully for this year. The best way to contribute is to sponsor the project which will accelerate development towards this feature.

**Reply by braum-me** [reply]
Hi @tommoor, any update on this? We would benefit from document/page approval workflows for our ISO requirements too.

**Reply by tommoor** [reply]
Ouch, hard to see that 2022 comment here. 

We're hearing ISO more and more and that's helping to get this prioritized. Right now just deciding if that is worthy of a specific feature or if it could be wrapped into a more general implementation of document properties.

**Reply by braum-me** [reply]
We're subject to ISO 9001, 14001, 27001 and a few more local or specific requirements like NIS2, TISAX and so on, and the common thread across all of them is controlled documents: review cycles, approvals, and a clear record of who signed off and when.
A general document properties approach with approval states and audit history would cover all of these at once. Happy to help shape the requirements together with our CISO, who's an absolute fan of Outline by the way.

**Reply by tommoor** [reply]
If you can email hello@getoutline.com with the requirements that would be very useful, in particular if the review cycle needs to be encoded in the platform
